tests/auth_tests/test_hashers.py

```diff
@@ -462,13 +490,12 @@ def setter():
     @override_settings(
         PASSWORD_HASHERS=[
             "django.contrib.auth.hashers.PBKDF2PasswordHasher",
-            "django.contrib.auth.hashers.SHA1PasswordHasher",
             "django.contrib.auth.hashers.MD5PasswordHasher",
         ],
     )
     def test_no_upgrade_on_incorrect_pass(self):
         self.assertEqual("pbkdf2_sha256", get_hasher("default").algorithm)
-        for algo in ("sha1", "md5"):
+        for algo in ("md5",):
```

Comment:
Same question.

Reply:
Maybe we could add `pbkdf2_sha1` in place of `sha1`.